dashboard/lib/pd/jot_form/constants.rb

```diff
@@ -10,14 +10,19 @@ module Constants
         TYPE_CHECKBOX = 'checkbox'.freeze,
         TYPE_SCALE = 'scale'.freeze,
         TYPE_MATRIX = 'matrix'.freeze,
+        TYPE_BUTTON = 'button'.freeze,
+        TYPE_DIVIDER = 'divider'.freeze,
+        TYPE_TEXT = 'text'.freeze
       ].freeze
 
       IGNORED_QUESTION_TYPES = [
-        TYPE_HEADING = 'head'.freeze,
+        TYPE_HEADING = 'heading'.freeze,
```

Comment:
? I'm fairly certain there is no `heading` type. There is a `head` type, which represents a heading. I had previously named the constant `TYPE_HEADING` for the `head` JotForm type, but I'm happy to be consistent and call it `TYPE_HEAD` if you prefer. In any case we shouldn't need both.

Reply:
I thought I was getting this as an ignored type - but if not then I'll remove it